quattor/functions/filesystem.pan

```diff
@@ -28,7 +28,7 @@ function boot_disk = {
       return(i);
     };
   };
-  undef;
+  null;
```

Comment:
I'm confused too 😄 Not sure what I did wrong but clearly was not the appropriate result... In fact, I realize that the problem was already fixed but that I had it (and thought it was not) because we are not up-to-date with the last version of the template library.... Closing this PR, sorry for the noise.